example_xdevice.py

```diff
@@ -27,6 +27,7 @@ def setup(self):
             min=0, max=1_000_000, step=1, _value=0.0
         ))
         self.add_property(nv)
+        self.wake_INDI_server()
```

Comment:
Here is an example of how to call the method to wake the MagAOX INDI server.

This call will do nothing if the INDI server control FIFO, ```/opt/MagAOX/drivers/fifos/indiserver.ctrl``` does not exist.